sidestep_engine/cli/validation.py

```
@@ -47,23 +47,73 @@ def validate_paths(args: argparse.Namespace) -> bool:
+                        max_sample_s = dur
+                    del _d
+                    if max_sample_s > 0 and max_sample_s < chunk_dur:
+                        break
+                if max_sample_s > 0 and chunk_dur > max_sample_s:
+                    print(
+                        f"[WARN] --chunk-duration {chunk_dur}s exceeds your longest sample "
+                        f"({max_sample_s:.0f}s). Chunking will have no effect.",
+                        file=sys.stderr,
+                    )
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)